### PR TITLE
Add more explanation for jump from part a to b

### PR DESCRIPTION
**Pull request by animeshsingh**

### CODE REVIEW

**Comment by animeshsingh** [comment]
@AnthonyAmanse take a quick look

File: README.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ This step shows you how to collect trace spans using [Zipkin](http://zipkin.io).
 
 ## Part B:  Modify sample application to use an external datasource, deploy the application and Istio envoys with egress traffic enabled
 
-#### In this part, we will modify the sample BookInfo application to use use an external database, and enable the Istio envoys for egress traffic. Please ensure you have the Istio control plane installed on your Kubernetes cluster as mentioned in the prerequisites. This part requires you to be in the root directory of this repository to use the YAML files and/or source code for the microservices. We will run these commands from the demo folder we cloned the source code repository in at the beginning.
+In this part, we will modify the sample BookInfo application to use use an external database, and enable the Istio envoys for egress traffic. Please ensure you have the Istio control plane installed on your Kubernetes cluster as mentioned in the prerequisites. This part requires you to be in the root directory of this repository to use the YAML files and/or source code for the microservices. We will run these commands from the demo folder we cloned the source code repository in at the beginning.
 
 ```bash
 cd demo
```
Comment (AnthonyAmanse) [inline README.md]: We'll no longer need to be in the demo folder. The next commands now reflect the "demo" folder. Ex: `kubectl apply -f demo/mysql-data`

Comment (Tomcli) [inline README.md]: - we no longer need `cd demo` since we are using `kubectl apply -f demo/...`

File: README.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ This step shows you how to collect trace spans using [Zipkin](http://zipkin.io).
 
 ## Part B:  Modify sample application to use an external datasource, deploy the application and Istio envoys with egress traffic enabled
 
-#### In this part, we will modify the sample BookInfo application to use use an external database, and enable the Istio envoys for egress traffic. Please ensure you have the Istio control plane installed on your Kubernetes cluster as mentioned in the prerequisites. This part requires you to be in the root directory of this repository to use the YAML files and/or source code for the microservices. We will run these commands from the demo folder we cloned the source code repository in at the beginning.
+In this part, we will modify the sample BookInfo application to use use an external database, and enable the Istio envoys for egress traffic. Please ensure you have the Istio control plane installed on your Kubernetes cluster as mentioned in the prerequisites. This part requires you to be in the root directory of this repository to use the YAML files and/or source code for the microservices. We will run these commands from the demo folder we cloned the source code repository in at the beginning.
```
Comment (Tomcli) [inline README.md]: `This part requires you to be in the root directory of this repository` is not correct, the user should be in the ibm directory.

**Comment by paulczar** [comment]
Can we clarify if the user is supposed to start with a fresh kubernetes install, or if they should run the commands on top of the already deployed versions of the book app ?

**Comment by Tomcli** [comment]
LGTM

**Pull request merged**